int16 delta_x
int16 delta_y
int16 scale
int16 status
